### PR TITLE
[Wallet] Add app language on start analytics event

### LINKED ISSUE #7003: Send app language on app_launched analytics event
### Expected Behavior

The `app_launched` analytics event includes the language the user uses their app on.

### Current Behavior

It doesn't include such field.

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Add `language` to the `app_launched` analytics event which represents the language the app is being used in.

### Tested

Manually

### Related issues

- Fixes #7003

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/app/App.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class App extends React.Component<Props> {
       deviceWidth: width,
       reactLoadDuration,
       appLoadDuration,
+      language: i18n.language,
```
Comment (tarikbellamine) [inline packages/mobile/src/app/App.tsx]: Sometimes `i18n` has weird initialization issues that cause random crashes and stuff. Does pulling language in this way have the same net effect as pulling from the Redux store?

**Review by tarikbellamine — APPROVED** [review]

File: packages/mobile/src/app/App.tsx (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/app/App.tsx]: I though about this, `i18n.language` has been null at times, yes, but that is most likely when the app is opened for the first time and this code is run before the store initialization, so language is probably going to be empty there as well. I'm not sure which one is best, maybe I can add one as fallback for the other

Comment (gnardini) [reply packages/mobile/src/app/App.tsx]: Added a fallback on the store value. On my tests, both are not null, but there might be edge cases, so I think having both will be probably good.

**Review by etuleu — APPROVED** [review]
